util.js

```diff
@@ -0,0 +1,32 @@
+const mapLimits = require('./conf.json')
+
+const mapX = mapLimits.width
+const mapY = mapLimits.height
+
+class Util {
+  static findFreeCell (players) { // Returns the coordinates of the free cell (Will go into an infinite loop if the whole field is filled with players)
+    let spawnX, spawnY
+    do {
+      spawnX = Util.randomNumber(mapX)
+      spawnY = Util.randomNumber(mapY)
+    } while (Util.isCellOccupied(spawnX, spawnY, players) !== false)
```

Comment:
Pretty interesting syntax you've got there, didn't even know that was possible in JavaScript 🤔

Reply:
Yeah, it's possible. I was thinking of using recursion, but it should be easier to read.